app/adapters/lot.js

```diff
@@ -49,6 +49,7 @@ const LotColumnsSQL = [
 
 export const cartoQueryTemplate = function(id) {
   return `SELECT ${LotColumnsSQL.join(',')},
+    /* id:${id} */
```

Comment:
Can you explain why you're removing this? If it really should be removed, let's just delete the line instead of commenting

Reply:
Don't feel like I _totally_ understand what's going on here but if you're confident that this line isn't affecting anything outside of when tests are running, I'm happy to defer to your Ember expertise here.